### PR TITLE
Feature/xz 155/readme documentation

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,150 @@
...
+
+The XxllncZGWBundle backend codebase utilizes the Common Gateway as an open-source installation framework. This means that the XxllncZGWBundle library, in its core form, functions as a plugin on this Framework. To learn more about the Common Gateway, you can refer to the documentation [here](https://commongateway.readthedocs.io/en/latest/).
+
+The CommonGateway Admin UI frontend comes with te CommonGateway.
+
+To install the backend, follow the steps below:
+
+### Gateway Installation
+
+1.  If you do not have the Common Gateway installed, you can follow the installation guide provided [here](https://github.com/ConductionNL/commonground-gateway#readme). The Common Gateway installation is required for the backend setup. You can choose any installation method for the gateway, such as Haven, Kubernetes, Linux, or Azure, and any database option like MySQL, PostgreSQL, Oracle, or MsSQL. The gateway framework handles this abstraction.
```
Comment (WilcoLouwerse) [inline README.md:26]: https://github.com/ConductionNL/commonground-gateway#readme
^ This is fine, but f.y.i. currently, latest version of this readme is only present on the dev branch and not the main branch

File: README.md [hunk 2]
```diff
@@ -1,7 +1,150 @@
...
+
+### XxllncZGWBundle Installation - Admin-UI
```
Comment (WilcoLouwerse) [inline README.md:28]: A little while ago we (Robert, Sarai, and I) decided to call it the Gateway UI.
I think it is best to be consistent about this and call it "Gateway UI" everywhere, haven't had the chance to ask Ruben his opinion on this however

File: README.md [hunk 3]
```diff
@@ -1,7 +1,150 @@
...
+3.  Click on the "common-gateway/xxllnc-zgw-bundle" card and then click on the "Install" button to install the plugin.
+4.  The admin-ui allows you to install, upgrade, or remove bundles. However, to load all the required data (schemas, endpoints, sources), you need to execute the initialization command in a php terminal.
+
+### XxllncZGWBundle Installation - Terminal
+
+1.  Open a php terminal and run the following command to install the XxllncZGWBundle:
+
+        docker-compose exec php composer require common-gateway/xxllnc-zgw-bundle
+
+### Initialization Command (Terminal)
```
Comment (WilcoLouwerse) [inline README.md:41]: This is also fine,
but I do wonder if we mention this step in the CoreBundle -> Plugin.md docu as well and if we should repeat this here, probably yes.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,150 @@
...
+
+    To load all the data along with specific content (like testdata), run:
+
+        docker-compose exec php bin/console commongateway:initialize -data
+
+With these steps completed, the backend setup for the XxllncZGW project should be ready to use. If you encounter any issues during the installation process, seek assistance from the development team. Happy coding!
+
+## Admin UI - Setup Instructions
+
+Once the backend (and frontend) is up and running, the HuwelijksplannerBundle can be configured. To ensure proper functionality, the sources and Security Group (Default Anonymous user) need to be modified. Other adjustments are optional.
```
Comment (WilcoLouwerse) [inline README.md:57]: HuwelijksplannerBundle ^

File: README.md [hunk 5]
```diff
@@ -1,7 +1,150 @@
...
+        *   Go to `Settings` in the Admin UI.
+        *   Navigate to the `Users` tab.
+        *   Select the user and edit the password.
+
+2.  **Security Group**
+    *   Add the scopes for the Default Anonymous in the Security Group.
+        *   Go to `Settings` in the Admin UI.
+        *   Navigate to the `Security Groups` tab
+        *   Locate and select `Default Anonymous` to view its details
+        *   Add the following scopes under the `Scopes` section:
```
Comment (WilcoLouwerse) [inline README.md:72]: This step might be possible to add to the XxllncZGWBundle instead,
By adding a SecurityGroup with these scopes and updating the Default Anonymous user to use that SecurityGroup in the installation.json by using the default reference of that user 🤔

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,7 +1,150 @@
...
+1.  After successfully installing the Gateway, access the admin-ui and log in.
+2.  In the left menu, navigate to "Plugins" to view a list of installed plugins. If you don't find the "common-gateway/xxllnc-zgw-bundle" plugin listed here, you can search for it by clicking on "Search" in the upper-right corner and typing "xxllnc" in the search bar.
+3.  Click on the "common-gateway/xxllnc-zgw-bundle" card and then click on the "Install" button to install the plugin.
+4.  The admin-ui allows you to install, upgrade, or remove bundles. However, to load all the required data (schemas, endpoints, sources), you need to execute the initialization command in a php terminal.
+
+### XxllncZGWBundle Installation - Terminal
+
+1.  Open a php terminal and run the following command to install the XxllncZGWBundle:
+
+        docker-compose exec php composer require common-gateway/xxllnc-zgw-bundle
```
Comment (rjzondervan) [inline README.md:39]: The best way of showing a CLI command is:
```cli
$ docker compose exec php composer require common-gateway/xxllnc-zgw-bundle
```

which is written like this: 
````md
```cli
$ docker compose exec php composer require common-gateway/xxllnc-zgw-bundle
```
````

**Review by rjzondervan — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (bbrands02) [reply README.md:78]: Could be done in a different issue

**Review by WilcoLouwerse — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (WilcoLouwerse) [reply README.md:78]: https://conduction.atlassian.net/browse/XZ-161

**Pull request merged**